webextensions/api/tabs.json

```diff
@@ -2999,7 +2996,7 @@
               "__compat": {
                 "support": {
                   "chrome": {
-                    "version_added": "≤58"
+                    "version_added": "48"
```

Comment:
✅ https://github.com/chromium/chromium/commit/6db4ce4333e219abec7f2f9eee1e75a174769ea4 in [48.0.2545.0](https://github.com/chromium/chromium/releases/tag/48.0.2545.0) before 48.0.2564.